extensions/positron-python/src/client/common/installer/productService.ts

```diff
@@ -23,6 +23,7 @@ export class ProductService implements IProductService {
         this.ProductTypes.set(Product.python, ProductType.Python);
         // --- Start Positron ---
         this.ProductTypes.set(Product.fastapiCli, ProductType.DataScience);
+        this.ProductTypes.set(Product.sqlite3, ProductType.DataScience);
```

Comment:
This is not _really_ a data science package, but it is most like that given the current options in `ProductType`. I did try adding a new type but then we would need to make an installer for it and that doesn't seem important, given that we can't install the missing system requirement for people in a straightforward way.

Reply:
IIUC the names are misleading. This looks like the right place.